src/core/InputToggle/style.ts

```diff
@@ -0,0 +1,154 @@
+import styled from "@emotion/styled";
+import { Switch, SwitchProps } from "@material-ui/core";
+import {
+  CommonThemeProps,
+  fontBodyXs,
+  getBorders,
+  getColors,
+  getCorners,
+  getSpaces,
+} from "../styles";
+
+export interface ExtraProps extends SwitchProps, CommonThemeProps {
+  offLabel?: string;
+  onChange?(e: React.ChangeEvent): void;
+  onLabel?: string;
+}
+
+const toggle = (props: ExtraProps) => {
+  const { disabled } = props;
+  const corners = getCorners(props);
+  const spaces = getSpaces(props);
+  const TOGGLE_HEIGHT = 18;
+
+  return `
+    cursor: ${disabled ? "default" : "pointer"};
+    border-radius: ${corners?.l}px;
+    height: 26px;
+    width: 62px;
+    line-height: 18px;
```

Comment:
@chanzuckerberg/sds-design Are these values we'll reuse/should we define these as bases somewhere?

Reply:
Hmm I can imagine there are cases where we'd want the toggle to be able to increase the width if needed (like if we needed to use a longer word). We haven't really discussed that or how to handle hardcoded cases. 

Would we be able to allow products using this component to override the dimensions if needed?

Reply:
I believe comsumer could override with width, as written, by importing the sds toggle and then using emotion/styled.